classroom/snippets/snippets.py

```diff
@@ -14,12 +14,76 @@
 
 from __future__ import print_function
 from googleapiclient import errors
-
+import json
 
 class ClassroomSnippets(object):
     def __init__(self, service):
         self.service = service
```

Comment:
classroom/snippets/ is just this file right now right? Where is the service getting initialized?

Reply:
yep, only `snippets.py` exists there right now. when i talked to eric about this, i think he said for now it's not being initialized but that eventually we'd write tests for classroom snippets as you did with slides, so that i could go with this format for now